README.md

```diff
@@ -51,6 +51,37 @@ Run `make precommit` before commiting.
 If you are using PyCharm, please set your docstring format to "Google" and your unit test runner to "PyTest" in
 `Preferences | Tools | Python Integrated Tools`.
 
+# Suggested Reading
```

Comment:
Maybe add an introduction like: "Below are some papers which have informed our design choices and may be referenced in issues."